plugins/modules/rulebook_activation.py

```diff
@@ -53,6 +73,27 @@
     description:
       - The extra variables for the rulebook activation.
     type: str
+  restart:
+    description:
+      - Whether or not the activation should be restarted.
+    type: bool
+    default: false
+  restart_wait:
+    description:
+      - Whether the system should wait after an activation restart.
+    type: bool
+    default: false
+  restart_wait_time:
+    description:
+      - Represents the total number of seconds that the system waits,
+        after an activation restart, before it returns.
+    type: int
+    default: 200
+  restart_interval:
```

Comment:
The name `restart_interval` is misleading. It sounds like the interval between multiple restarts. It is actually the check status interval. To me user does not need to set the interval.